decidim-proposals/spec/services/decidim/proposals/proposal_serializer_spec.rb

```diff
@@ -47,6 +47,86 @@ module Proposals
           expect(serialized).to include(id: proposal.id)
         end
 
+        describe "author" do
+          context "when it is an official proposal" do
+            let!(:proposal) { create(:proposal, :official) }
+
+            before do
+              component.participatory_space.organization.update!(name: { en: "My organization" })
```

Comment:
I do not think that we have the internationalized organization in 0.28. 
```suggestion
              component.participatory_space.organization.update!(name: "My organization" )
```